igamt-lite-controller/src/main/java/gov/nist/healthcare/tools/hl7/v2/igamt/lite/web/controller/NotificationController.java

```diff
@@ -27,9 +62,44 @@ public Notifications findNotificationByIgId(@PathVariable("id") String id)
     return notificationsRepository.findByIgDocumentId(id);
   }
 
-
-  @RequestMapping(value = "/delete/{id}", method = RequestMethod.POST, produces = "application/json")
+  @RequestMapping(value = "/delete/{id}", method = RequestMethod.POST,
+      produces = "application/json")
   public void deleteNotification(@PathVariable("id") String id) throws DataNotFoundException {
     notificationsRepository.delete(id);
   }
+
+  @RequestMapping(value = "/phinvadsEmailNotification/{id}", method = RequestMethod.GET,
+      produces = "application/json")
+  public @ResponseBody boolean notifyEmailForPHINVADS(@PathVariable("id") String id)
+      throws Exception {
+    Notifications notifications = notificationsRepository.findOne(id);
+    if (notifications != null) {
+      IGDocument igDoc = igDocumentService.findById(notifications.getIgDocumentId());
+      if (igDoc != null) {
+        ExportConfig config = exportConfigService.findOneByAccountId(igDoc.getAccountId());
+        if(config != null && config.isPhinvadsUpdateEmailNotification()){
+          sendNotificationPhinvadsUpdateEmail(igDoc, notifications);
+          return true;
+        }
+      }
+    }
+    return false;
+  }
+
+  private void sendNotificationPhinvadsUpdateEmail(IGDocument doc, Notifications notis) {
+    SimpleMailMessage msg = new SimpleMailMessage(this.templateMessage);
+    Account targetAccount = accountRepository.findOne(doc.getAccountId());
+    msg.setSubject("NIST IGAMT IGDocument Share");
+    msg.setTo(targetAccount.getEmail());
+    msg.setText(
```

Comment:
Change the text to:
Dear XXX,
The following phinvads valuesets have been updated in your IG Document: XXXX 
- BindingIdentifier1: name  
- BindingIdentifier1: name  

etc...

If you have any question , please feel free to reach out at XXXX